tests/test_vidispine_api.py

```diff
@@ -139,15 +143,15 @@ def test_400(self):
         :return:
         """
         from gnmvidispine.vidispine_api import VSApi, VSBadRequest
-        request_body = """<MetadataDocument xmlns="http://xml.vidispine.com/">
+        request_body = b"""<MetadataDocument xmlns="http://xml.vidispine.com/">
```

Comment:
so request() _does_ expect a `bytestring` for the request body

Reply:
Yes.

Reply:
i would update to verify that the body is either None or a bytestring and raise a TypeError otherwise to make it more explicit

Reply:
Sorry, my answer was wrong or at least confusing.

Under Python 2 I think it expects a unicode object.

Under Python 3 I think is expects a str or bytes object.